pkg/parser/custom/cyclonedx_sbom/cyclonedx.go

```diff
@@ -1,18 +1,17 @@
-package parser
+package cyclonedx_sbom
```

Comment:
As per go convention, we should use a package name such as `sbomutils`

> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps

https://go.dev/doc/effective_go